Makefile

```diff
@@ -1,7 +1,7 @@
 GIT_TAG?=$(shell git rev-parse --short HEAD)
```

Comment:
Is this for experimental and/or official releases?

Because for official we should use version instead

Reply:
@tomkerkhove this is for experimental / dev releases. If someone wants to install an official release, we'd have them do a `helm install` directly.